memory-viz/src/tests/cli.spec.tsx

```diff
@@ -64,11 +49,9 @@ describe.each([
         exec(`memory-viz ${command}`, (err) => {
             if (err) throw err;
 
-            const svgFilePath = getSVGPath(command.includes("--output"));
-            const fileContent = fs.readFileSync(svgFilePath, "utf8");
-
+            const fileContent = fs.readFileSync(outputPath, "utf8");
```

Comment:
It's more robust to keep the `.includes("--output")`: this does not assume that `--output` is always set to `outputPath`.